### PR TITLE
Design system colors

### PR DESCRIPTION
**Pull request by flozia**
This adds the [Nebula Design System Colors](https://www.figma.com/file/cBaBmcPisqZ0QQX5LLzpjU/Nebula-Design-System-V1?node-id=35%3A0).

┆Issue is synchronized with this [Jiraserver Bug](https://mozilla-hub.atlassian.net/browse/VPN-817)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,6 +58,7 @@ macos/gobridge/wireguard-go-version.h
 .vscode/
 MozillaVPN.xcodeproj/
 MozillaVPN.vcxproj*
+*.autosave
```
Comment (flozia) [inline .gitignore:61]: Not sure if this change should be in this PR since it is not related.

Comment (lesleyjanenorton) [reply .gitignore:61]: It's a good addition though!

File: src/ui/themes/colors.js [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+ * Helper functions
+ */
+const percentToHex = percent => {
+  const int8Bit = Math.round(percent * 255);
+  const hexAlphaValue = int8Bit.toString(16).padStart(2, '0');
+
+  return hexAlphaValue.toUpperCase();
+};
+
+const addTransparency = (hexColor, percent) => {
```
Comment (lesleyjanenorton) [inline src/ui/themes/colors.js:130]: This is so cool

**Review by lesleyjanenorton — APPROVED** [review]
Thanks @flozia! This is awesome and will be so useful going forward.

**Pull request merged**